service/javadsl/server/src/main/java/com/lightbend/lagom/javadsl/server/ServiceGuiceSupport.java

```diff
@@ -3,23 +3,43 @@
  */
 package com.lightbend.lagom.javadsl.server;
 
-import com.lightbend.lagom.internal.javadsl.*;
+import com.google.inject.Binder;
+import com.lightbend.lagom.internal.javadsl.BinderAccessor;
+import com.lightbend.lagom.internal.javadsl.server.JavadslServicesRouter;
 import com.lightbend.lagom.internal.javadsl.server.ResolvedServices;
 import com.lightbend.lagom.internal.javadsl.server.ResolvedServicesProvider;
 import com.lightbend.lagom.internal.javadsl.server.ServiceInfoProvider;
-import com.lightbend.lagom.internal.javadsl.server.JavadslServicesRouter;
 import com.lightbend.lagom.internal.server.status.MetricsServiceImpl;
-
-import com.lightbend.lagom.javadsl.server.status.MetricsService;
-import com.google.inject.Binder;
 import com.lightbend.lagom.javadsl.api.ServiceInfo;
 import com.lightbend.lagom.javadsl.client.ServiceClientGuiceSupport;
+import com.lightbend.lagom.javadsl.server.status.MetricsService;
+
+import java.util.Arrays;
 
 public interface ServiceGuiceSupport extends ServiceClientGuiceSupport {
 
+    // redefines bindServiceInfo from super interface.
```

Comment:
Both `ServiceGuiceSupport` and `ServiceClientGuiceSupport` are undocumented at code level. All  docs are provided in the documentation. Will fix in a separate PR.

This is an implementors note because redefining a `default` doesn't require a keyword or annotation so it is not obvious that an override is happening. A developer altering the behaviour of `ServiceGuiceSupport#bindServiceInfo` will only see that it's a method that overrides a parent interface through this comment.